MaxText/configs/v5p/32b.sh

```diff
@@ -6,10 +6,9 @@ echo "Running 32b.sh"
 # OUTPUT_PATH (Required, unless base_output_directory is already set in base.yml)
 # DATASET_PATH (Required, unless dataset_path is already set in base.yml)
 # RUN_NAME (Required, unless run_name is already set in base.yml or running with XPK/GKE)
-# PLATFORM (Optional, can be "gke" or "gce", default is "gce")
```

Comment:
Sorry one last thing, could you take out the `export PLATFORM="gce"` in this file on line 20?